### ISSUE TITLE
ng-drag-handle is not working

### ISSUE THREAD

**Issue by confa** [fatlinesofcode/ngDraggable#213]
var dragHandles = element.find('[ng-drag-handle]');"

JLite 'find' calls findElementsByTagName under the hood, thus it will never find [ng-drag-handle] attribute.

"angular": "1.4.7",
"ngDraggable": "0.1.8",

**Comment by delmotte** [comment]
change :
var dragHandles = element.find('[ng-drag-handle]');
by :
var dragHandles = angular.element(element[0].querySelectorAll('[ng-drag-handle]'));

line 48

**Comment by confa** [comment]
Yeah, I've done this. Do you want me to make a pull request for this? You know, it's not as that good to have a custom build for library.

**Comment by delmotte** [comment]
Yes i will do it when i'll have time :)

**Comment by kresli** [comment]
is there a pull request for this?
